examples/bootstrapping/vagrant/onboarding/README.md

```diff
@@ -0,0 +1,117 @@
+accompanying [Vagrantfile](./client/Vagrantfile). Note that before you provision either of these machines you will need to create a `settings.rb` file from the `settings.rb.EXAMPLE` in both the `client/` and `server/` directories. You will then need to adjust the value of the `SHARED_DIRECTORY` variable in each file to match the location of where you checked out this repository on your machine. This is important because both the server and client you are configuring will use this directory to created and consume key and settings files used to configure the two machines for this exercise. 
+
+After you've provisioned each machine you will then use the address you added to your `/etc/hosts` or HOSTS file to access Chef Automate through your browser using the credentials found in the `automate-credentials.toml` file in the `SHARED_DIRECTORY` location. This credentials file was generated during the `server` bootstrap process. 
+
+## Before You Start
+
+### Assumptions
+
+We assume the following:
+
+- You have unrestricted access to the Internet.
+- You have a computer capable of hosting virtual machines  (e.g. at least two cores, at least 8GB of free RAM and at least 60 GB of disk available).
+- You have administrative rights to install software on your computer.
+
+### Prerequisites
+
+We assume you have the following software installed and/or available to you:
+
+- The latest version of [Virtual Box](https://www.virtualbox.org/wiki/Downloads) is installed.
+- The latest version of [Vagrant](https://www.vagrantup.com/downloads) is installed.
+- You have access to the latest `generic` vagrant box images or an unrestricted connection to 
+  the Internet to download the boxes.
+- You've successfully run the command `vagrant box add` with the distribution you want to use 
+  for the exercises described in this README.
+  - All free `generic` boxes can be found through [vagrantup](https://app.vagrantup.com/boxes/search) or
+    in the following list:
+
+    - `generic/rhel7`
+    - `generic/rhel8`
+    - `generic/centos7`
+    - `generic/ubuntu1804`
+- You plan to use the Ubuntu 18.04 version as that was the only version tested in this process. Please submit a PR once you've successfully tested against other distributions or if you'd like to add directions for Windows.
+
+## Provision Chef Automate with Chef Infra Server 
+
+The first step we're taking in this exercise is to install Chef Automate with Chef Infra Server using Vagrant. To accomplish this task you will need to do the following:
+
+* Change the `server` directory.
+* Update the `Vagrantfile` with the `BOX` you want to use.
+  * For example an Ubuntu box, `BOX=generic/ubuntu1804`
+* Run `vagrant up`
```

Comment:
Add a step before vagrant up to update the settings.rb, also assuming the user needs to remove the .example?